squid-blockexplorer/src/processor.ts

```diff
@@ -9,39 +9,30 @@ import { Store, TypeormDatabase } from '@subsquid/typeorm-store'
 import config from './config'
 import { getErrorMessage } from './utils'
 import { processBalances } from './balances'
+import { processBlocks } from './blocks'
 
 export type Item = BatchProcessorItem<typeof processor>
 export type EventItem = BatchProcessorEventItem<typeof processor>
 export type CallItem = BatchProcessorCallItem<typeof processor>
 export type Context = BatchContext<Store, Item>
 
-const addEventDataConfig = {
-    data: { event: { args: true } },
-} as const
-
 const processor = new SubstrateBatchProcessor()
     .setBatchSize(config.batchSize || 500)
     .setDataSource(config.dataSource)
     .setBlockRange(config.blockRange || { from: 0 })
-    .addEvent('Balances.Endowed', addEventDataConfig)
-    .addEvent('Balances.Transfer', addEventDataConfig)
-    .addEvent('Balances.BalanceSet', addEventDataConfig)
-    .addEvent('Balances.Reserved', addEventDataConfig)
-    .addEvent('Balances.Unreserved', addEventDataConfig)
-    .addEvent('Balances.ReserveRepatriated', addEventDataConfig)
-    .addEvent('Balances.Deposit', addEventDataConfig)
-    .addEvent('Balances.Withdraw', addEventDataConfig)
-    .addEvent('Balances.Slashed', addEventDataConfig)
+    .addEvent('*') // process all events
+    .addCall('*') // process all calls
```

Comment:
did you notice a significant change in processing time after making these changes?

Reply:
yes, I had to decrease batch size to 20 blocks instead of 50, otherwise I was having timeout issues